rst2ctags.py

```diff
@@ -114,8 +116,8 @@ def findSections(filename, lines):
             else:
                 topLine = ''
 
-            # If the heading line is to short, then docutils doesn't consider it
-            # a heading.
+            # If the heading line is to short, then docutils doesn't consider
+            # it a heading.
```

Comment:
Please don't change lines that are not affected by your patch.  The comment, as it existed before, fit my coding standard.